dali/operators/reader/loader/video_loader.h

```diff
@@ -163,13 +163,20 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
       codec_id_(0),
       skip_vfr_check_(spec.GetArgument<bool>("skip_vfr_check")),
       file_list_frame_num_(spec.GetArgument<bool>("file_list_frame_num")),
+      file_list_include_preceding_frame_(spec.GetArgument<bool>(
+                                                          "file_list_include_preceding_frame")),
       pad_sequences_(spec.GetArgument<bool>("pad_sequences")),
       stats_({0, 0, 0, 0, 0}),
       current_frame_idx_(-1),
       stop_(false) {
     DALI_ENFORCE(stride_ > 0, "Stride should be > 0");
     if (step_ < 0)
       step_ = count_ * stride_;
+    if (!file_list_include_preceding_frame_) {
+      DALI_WARN("`file_list_include_preceding_frame_` is set to false (or not set at all). In "
+                "future releases, the default behavior would be changed to True and the argument "
+                "will be removed.");
```

Comment:
Adding mandatory argument that will be removed in future releases is a pathway to the code compatibility issues between releases.

Reply:
Maybe we can change the default first and then remove?

Reply:
The main point is, that this creates an opportunity to write code, that will be at that point the way advertised by us (you need to set `file_list_include_preceding_frame_` to True), but removing that argument will lead to it not working in the future (error on setting argument that doesn't exist). I'm not convinced this is the best way of doing stuff.

Reply:
Done as discussed